sdk/nodejs/provisioning/bin.ts

```diff
@@ -132,12 +154,20 @@ export class Bin implements EngineConn {
   ): Promise<Client> {
     const args = [binPath, "session"]
 
-    if (opts.Workdir) {
-      args.push("--workdir", opts.Workdir)
-    }
-    if (opts.Project) {
-      args.push("--project", opts.Project)
-    }
+    const sdkVersion = await this.getSDKVersion()
+
+    const flagsAndValues = [
+      { flag: "--workdir", value: opts.Workdir },
+      { flag: "--project", value: opts.Project },
```

Comment:
I think we should continue to set `--workdir` and `--project` only if they've been set by the user. This allows consistent use of defaults from CLI instead of SDK.